### PR TITLE
Deployed/demo manifests for failed samples

### PR DESCRIPTION
**Pull request by Keshav-MSFT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,26 +18,27 @@
 1. [Scenario specific samples](#Scenario-specific-samples)
 
 
-## Try it yourself - experience the Apps in your Microsoft Teams client
+## Try it yourself - experience the Apps in your Microsoft Teams client.
 
 Here are all the samples which are deployed on Microsoft Azure and you can try it yourself by uploading respective app packages (.zip files links below) to one of your teams and/or as a personal app. (Sideloading must be enabled for your tenant; [see steps here](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading).) These apps are running on the free Azure tier, so it may take a while to load (warm up) if you haven't used it recently and it goes back to sleep quickly if it's not being used. Rest assured, once it's loaded it's pretty snappy.
 
+Note: With sample Name added * Admin consent follow the links (https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/grant-admin-consent?pivots=portal).
```
Comment (Pawan-MSFT) [inline README.md:25]: Note: Need admin consent which samples name added with asterisk (*). Please follow the **links** to provide admin consent.
Link should be clickable.

File: README.md [hunk 2]
```diff
@@ -18,26 +18,27 @@
...
 |:--:|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:--------|
 |1|Task Module Demo       | Sample app showing off the Teams Task Module, a way to invoke custom code from a bot, a tab, or both |[View](/demo%20manifest/Task-Module.zip)     |
 |2|Meeting Recruitment App | Sample app showing meeting app experience for interview scenario. |[View](/demo%20manifest/Meeting-Recruitment-App.zip)     |
 |3| Product inspection | Demonstrating a feature where user can scan a product and mark it as approved/rejected. |[View](/demo%20manifest/Tab-Product-Inspection.zip)     |
 |4|Bot daily task reminder | This sample demos a feature where user can schedule a recurring task and get a reminder on the scheduled time|[View](/demo%20manifest/Bot-Daily-Task-Reminder.zip)     |
-|5| Message reminder setup through messaging extension | This sample demos a feature where user can schedule a task from messaging extension action command and get a reminder card at scheduled date and time.|[View](/demo%20manifest/Message-Reminder-Messaging-extension.zip)     |
```
Comment (Pawan-MSFT) [inline README.md:32]: why some sample missing ?? like "Message reminder setup through messaging extension, Meetings Extensibility, Meetings-ContextMeetings app"

Comment (Pawan-MSFT) [reply README.md:32]: Please make sure all working samples should be available.

**Review by Pawan-MSFT — COMMENTED** [review]
Please check all comments

File: README.md (hunk 2 above)
Comment (Keshav-MSFT) [reply README.md:32]: Message-Reminder-Messaging-extension.zip file not available as well as your excel sheet and meetings Extensibility(meeting token app) 33 number in main readme. and meeting context app available in this 39 number in readme.

File: README.md (hunk 1 above)
Comment (Keshav-MSFT) [reply README.md]: resolved

**Review by Wajeed-msft — APPROVED** [review]

**Review by Pawan-MSFT — APPROVED** [review]
looks correct approving it.

**Pull request merged**